akka-remote/src/main/resources/reference.conf

```diff
@@ -937,3 +651,313 @@ akka {
+       # if it does not update its C'n'C timestamp.
+       driver-timeout = 20 seconds
+
+       flight-recorder {
+         // FIXME it should be enabled by default when we have a good solution for naming the files
+         enabled = off
+         # Controls where the flight recorder file will be written. There are three options:
+         # 1. Empty: a file will be generated in the temporary directory of the OS
+         # 2. A relative or absolute path ending with ".afr": this file will be used
+         # 3. A relative or absolute path: this directory will be used, the file will get a random file name
+         destination = ""
+       }
+
+       # compression of common strings in remoting messages, like actor destinations, serializers etc
+       compression {
+
+         actor-refs {
+           # Max number of compressed actor-refs
+           # Note that compression tables are "rolling" (i.e. a new table replaces the old
+           # compression table once in a while), and this setting is only about the total number
+           # of compressions within a single such table.
+           # Must be a positive natural number.
+           max = 256
+
+           # interval between new table compression advertisements.
+           # this means the time during which we collect heavy-hitter data and then turn it into a compression table.
+           advertisement-interval = 1 minute # TODO find good number as default, for benchmarks trigger immediately
+         }
+         manifests {
+           # Max number of compressed manifests
+           # Note that compression tables are "rolling" (i.e. a new table replaces the old
+           # compression table once in a while), and this setting is only about the total number
+           # of compressions within a single such table.
+           # Must be a positive natural number.
+           max = 256
+
+           # interval between new table compression advertisements.
+           # this means the time during which we collect heavy-hitter data and then turn it into a compression table.
+           advertisement-interval = 1 minute # TODO find good number as default, for benchmarks trigger immediately
+         }
+       }
+
+       # List of fully qualified class names of remote instruments which should
+       # be initialized and used for monitoring of remote messages.
+       # The class must extend akka.remote.artery.RemoteInstrument and
+       # have a public constructor with empty parameters or one ExtendedActorSystem
+       # parameter.
+       # A new instance of RemoteInstrument will be created for each encoder and decoder.
+       # It's only called from the stage, so if it dosn't delegate to any shared instance
+       # it doesn't have to be thread-safe.
+       # Refer to `akka.remote.artery.RemoteInstrument` for more information.
+       instruments = ${?akka.remote.artery.advanced.instruments} []
+      }
+    }
+  }
+
+}
+#//#artery
```

Comment:
line break here is important (when merging of files for sbt assembly or similar)